nodestream_akamai/akamai_utils/cloudlets_v2_client.py

```diff
@@ -78,7 +95,7 @@ def extract_active_akamai_redirect_policy_versions(self, policy_tree):
         if len(list_of_activations) == 0:
             logger.info(
                 "Policy had no active occurrences. Candidate for cleanup: Policy ID %s",
-                policy_tree["policyId"],
+                str(policy_tree["policyId"]),

```

Comment:
The `%s` formatter should format the number without needing to call `str`. 

```suggestion
                policy_tree["policyId"],
```